coc-settings.json

```diff
@@ -39,9 +48,16 @@
     "python.linting.enabled": true,
     "python.linting.pylintEnabled": false,
     "python.linting.flake8Enabled": true,
+    "python.linting.flake8Args": ["--max-line-length", "88"],
     "python.analysis.logLevel": "Warning",
     "python.formatting.provider": "black",
-    "python.formatting.blackArgs": ["--line-length", "80"],
+    "python.formatting.autopep8Args": [
```

Comment:
This should be in a user configurable setting (preferably in a project `.coc-settings.json` file. Default is black, but changing it to `autopep8` might make sense in other projects or not.

Reply:
same as before, I did it here because I like the idea of use as default 88 🤔  as default for the projects but yeah we could add it in .coc-settings.json by project